src/neptune/api.d

```diff
@@ -0,0 +1,166 @@
+/*******************************************************************************
+
+    helper methods to access the octod/github API
+
+    Copyright:
+        Copyright (c) 2017 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module neptune.api;
+
+import octod.core;
+import octod.api.repos;
+
+
+/*******************************************************************************
+
+    Returns:
+        global api object to interact with github
+
+*******************************************************************************/
+
+public ref HTTPConnection api ( )
```

Comment:
I don't think using a singleton-like entity here is a wise call. Can you simply create it in `main` and store pointer where necesary?